File: app/src/main/java/com/easycopy/utils/StringUtils.java
package com.easycopy.utils;

/**
 * @author pankaj
 * @version 1.0
 * @since 2020-06-03
 */
public class StringUtils {

    public static boolean isEmpty(final CharSequence cs) {
        return cs == null || cs.length() == 0;
    }
}

File: presentation/src/main/java/com/easycopy/utils/StringUtils.kt
package com.easycopy.utils

/**
 * @author pankaj
 * @version 1.0
 * @since 2020-06-03
 */
object StringUtils {
    fun isEmpty(cs: CharSequence?): Boolean {
        return cs == null || cs.length == 0
    }
}